frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/DataConnectionExistingField.tsx

```diff
@@ -18,18 +28,35 @@ const DataConnectionExistingField: React.FC<DataConnectionExistingFieldType> = (
   dataConnections,
 }) => {
   const [isOpen, setOpen] = React.useState(false);
+  const isDataConnectionsEmpty = filterOutConnectionsWithoutBucket(dataConnections).length === 0;
+
   return (
     <Stack hasGutter>
       <StackItem>
+        {dataConnections.length !== 0 && (
+          <Popover
+            aria-label="Basic popover"
```

Comment:
the aria-label should be more specific, something like `No bucket popover`, you can have a look at the other aria-labels for reference.